judges/github-events/github-events.rb

```diff
@@ -93,8 +97,7 @@ def self.fill_up_event(fact, json)
       end
 
     when 'PullRequestReviewEvent'
-      case json[:payload][:action]
-      when 'created'
+      if reviewer?(json) && json[:payload][:action] == 'created'
```

Comment:
@Suban05 this doesn't sound like a solution. I believe, we should simply read `who` from `json[:payload][:review][:user]`

Reply:
@yegor256 I added two examples of JSON, which we fetch from GitHub.
Here is the first example
```
payload: {
  action: 'created',
  review: {
    id: 2_210_067_609,
    node_id: 'PRR_kwDOL6GCO86DuvSZ',
    user: {
      login: 'yegor256',
      id: 42,
      node_id: 'MDQ6VXNlcjUyNjMwMQ==',
      type: 'User'
    },
}
```
Here the reviewer is a user with id = 42
But in the second example, we see:

```
payload: {
  action: 'created',
  review: {
    id: 2_210_067_609,
    node_id: 'PRR_kwDOL6GCO86DuvSZ',
    user: {
      login: 'test',
      id: 526_200,
      node_id: 'MDQ6VXNlcjUyNjMwMQ==',
      type: 'User'
    }
}    
```          
Here `json[:payload][:review][:user]` is not the reviewer, because if we look at who the author of the PR is, we will see:

```
pull_request: {
  url: 'https://api.github.com/repos/yegor256/judges/pulls/93',
  id: 1_990_323_142,
  node_id: 'PR_kwDOL6GCO852oevG',
  number: 93,
  state: 'open',
  locked: false,
  title: 'allows to push gizpped factbase',
  user: {
    login: 'test',
    id: 526_200,
    node_id: 'MDQ6VXNlcjE2NDYwMjA=',
    type: 'User',
    site_admin: false
  }
}
```
I thought we had to skip examples like the second one.